src/patternfly/components/Switch/switch.scss

```diff
@@ -155,7 +167,6 @@
 
 .pf-c-switch__label {
   display: inline-block;
-  grid-column: 2;
```

Comment:
I think we talked about leaving the existing CSS in place so as not to break something someone might be relying on the existing CSS as it is currently. I believe it was something like this below, please correct me if I'm wrong!

```
.pf-c-switch {
  &.pf-m-reverse {
    .pf-c-switch__label,
    .pf-c-switch__toggle {
      grid-row: 1;
    }
    .pf-c-switch__label {
      grid-column: 1;
    }
  }
}
```

Reply:
Yes @mcoker, you are correct, patched.